google/cloud/spanner/samples/samples.cc

```diff
@@ -143,6 +143,13 @@ void DropDatabase(std::vector<std::string> const& argv) {
   (argv[0], argv[1], argv[2]);
 }
 
+google::cloud::spanner::Client MakeSampleClient(
+    std::vector<std::string> const& argv) {
```

Comment:
Given this requires an `argv.size()` of (at least) three, and it is the caller who actually checked that, perhaps `MakeSampleClient()` should be taking the three arguments separately?

Reply:
Done.